lib/config_read.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push('fluentd/lib')
+
+require 'fluent/config/v1_parser'
+require 'optparse'
+
+# Accepts a file path and prints out parsed version
+class ConfigFile
+  def initialize(argv = ARGV)
+    @argv = argv
+    prepare_option_parser
+    parse_options
```

Comment:
I think parsed configuration should be the return of this function.

The current way makes it hard to know where @file_parse is created as an instance variable.

Reply:
Good point, will make that change.